templates/pytorch/requirements.txt

```diff
@@ -0,0 +1,5 @@
+algorithmia>=1.0.0,<2.0
+six
+numpy
+Pillow
```

Comment:
We might want to version peg this.

Reply:
fyi - if you don't version peg then if a version is already installed pip won't install a new one (which might be preferable since we will have _some_ version installed in the base pytorch image which won't ever change).  i'm kind of counting on you guys to know what pip packages need to be installed for those as well

Reply:
I'd rather have all these package be as explicit as possible.

We want to be able to reproduce things if needed.

@zeryx: I would just do a `pip freeze` to get the version that was installed on your machine when you tested this locally.

Reply:
But should we not be using the  package that was bundled into the package set (if one exists) rather then pulling something else ?  So if this python package already exists (and we know that it does) why specify it in requirement.txt at all?

Reply:
yeah that was my point as well - the package (which doesn't yet exist but needs to in `langpacks/libraries/pytorch-1.0`) would do some `pip install` in there and that is the base image from which any algorithm would be built.  if there are conflicting versions set in requirements.txt then the space saving we get from the base image would be lost